File: common/src/main/kotlin/common/models/Route.kt
package common.models

data class Route(
    val fullLineCode: String,
    val simpleLineCode: String,
    val routeName: String,
    val codMode: String,
)

data class RouteWithItineraries(
    val fullLineCode: String,
    val simpleLineCode: String,
    val routeName: String,
    val codMode: String,
    val itineraries: List<Itinerary>,
)

File: MadridTransporte.Api/Data/Entities/Route.cs
namespace MadridTransporte.Api.Data.Entities;

public class TransitRoute
{
    public required string FullLineCode { get; set; }
    public required string SimpleLineCode { get; set; }
    public required string RouteName { get; set; }
    public required string CodMode { get; set; }
}
